Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gliderlabs/alpine
+
+COPY requirements.txt /opt/elodie/requirements.txt
+COPY docs/requirements.txt /opt/elodie/docs/requirements.txt
+COPY elodie/tests/requirements.txt /opt/elodie/elodie/tests/requirements.txt
+WORKDIR /opt/elodie
+
+RUN apk add --update boost-python ca-certificates exiftool exiv2 make python py-pip && \
```

Comment:
Unfortunately removing them at later steps doesn't reduce the size of the final image, because the previous layers still contribute to the size of the final image (unless you export and import it again to flatten the layers, in which case caching doesn't work anyway).

In looking at it some more tonight, the removal of the LatLon requirement after I created this Dockerfile means I can split the pip install out into a separate step, so that's nice!